modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,123 @@
+
+. To generate the manifests, run the following command from the directory that contains the installation program:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
+----
+
+. From the directory that contains the installation program, obtain the {product-title} release image that your `openshift-install` binary is built to use:
++
+[source,terminal]
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
+. Extract the `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal]
+----
+$ oc adm release extract --cloud=ibmcloud --credentials-requests $RELEASE_IMAGE \
+    --to=<path_to_credential_requests_directory> <1>
+----
+<1> The directory where the credential requests will be stored.
++
+This command creates a YAML file for each `CredentialsRequest` object.
++
+.Sample `CredentialsRequest` object
++
+[source,yaml]
+----
+  apiVersion: cloudcredential.openshift.io/v1
+  kind: CredentialsRequest
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: openshift-image-registry-ibmcos
+    namespace: openshift-cloud-credential-operator
+  spec:
+    secretRef:
+      name: installer-cloud-credentials
+      namespace: openshift-image-registry
+    providerSpec:
+      apiVersion: cloudcredential.openshift.io/v1
+      kind: IBMCloudProviderSpec
+      policies:
+      - attributes:
+        - name: serviceName
+          value: cloud-object-storage
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+        - crn:v1:bluemix:public:iam::::role:Operator
+        - crn:v1:bluemix:public:iam::::role:Editor
+        - crn:v1:bluemix:public:iam::::serviceRole:Reader
+        - crn:v1:bluemix:public:iam::::serviceRole:Writer
+      - attributes:
+        - name: resourceType
+          value: resource-group
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+----
+
+. Create the service ID for each credential request, assign the policies defined, create an API key in IBM Cloud, and generate the secret:
++
+[source,terminal]
+----
+$ ccoctl ibmcloud create-service-id \
+    --credentials-requests-dir <path_to_store_credential_request_templates> \ <1>
+    --name <name> \ <2>
+    --resource-group-name <resource_group_name> \ <3>
+    --output-dir <installation_directory>
+----
+<1> The directory where the credential requests are stored.
+<2> The name of the {product-title} cluster.
+<3> Optional: The resource group name of the cluster. This lets you narrow down the scope of the credential within the account.
++
+--
+[NOTE]
+====
+If an incorrect resource group name is provided, the installation fails during the bootstrap phase. To find the correct resource group name, run the following command:
+
+[source,terminal]
+----
+$ cat <installation_directory>/manifests/cluster-infrastructure-02-config.yml | grep resourceGroupName: | cut -d':' -f2
+----
+====
```

Comment:
Hi @codyhoag, I've double checked the whole content again as requested and it looks good to me in general, however, IMO, this section is still cumbersome from customer side, how about the following improvements?
- Moving the parameter `--resource-group-name` to the end of the example in correlation with the optional note as follows, on this way you can copy the rest and skip the optional part if needed:
~~~
$ ccoctl ibmcloud create-service-id \
    --credentials-requests-dir <path_to_store_credential_request_templates> \ <1>
    --name <name> \ <2>
    --output-dir <installation_directory> \
    --resource-group-name <resource_group_name> <3>
~~~
- Change the optional note as follows based on `ccoctl ibmcloud create-service-id -h`:
~~~
 <3> Optional: The name of the resource group used for scoping the access policies.
~~~
- And finally, the extra note section with the `cat` example I would prefer to simplify it as follows for clarity:
~~~
grep resourceGroupName <installation_directory>/manifests/cluster-infrastructure-02-config.yml
~~~
Regards

Reply:
Absolutely! I have updated those parts :+1: